slack-creds-extractor-main/native-host/save_tokens.py

```
@@ -0,0 +1,66 @@
+
+        token_file.write_text(xoxc_token)
+        cookie_file.write_text(d_cookie)
+
+        os.chmod(token_file, 0o600)
+        os.chmod(cookie_file, 0o600)
+
+        send_message({
+            "success": True,
+            "message": f"Tokens saved to {CONFIG_DIR}/",
+        })
+    except Exception as e:
```

Ruff (0.15.15)

[warning] Do not catch blind exception: `Exception`

(BLE001)